bears/jinja2/Jinja2Bear.py

```diff
@@ -112,11 +112,17 @@ class Jinja2Bear(LocalBear):
 
     VARIABLE_REGEX = re.compile(
         r'(?P<open>{{)(?P<content>.*?)(?P<close>}})')
+    STATEMENT_REGEX = re.compile(
+        r'(?P<open>{%[+-]?)'
+        r'(?P<content>(?!for)'
+        r'(?!if)(?!endfor)'
+        r'(?!endif).*?)'
```

Comment:
I think it would be good for the sake of readability to do it like
```python
r'(?P<open>{%[+-]?)'
r'(?P<content>'
r'(?!for)(?!if)(?!endfor)(?!endif)'
r'.*?)'
```
Is it also possible to use the `|` operator?
```
(?!for|if|endfor|endif)
```

Reply:
Nope this did not work. I tried it.

Reply:
`!` needs to applied for each one separately.